### PR TITLE
Release v5.5.0-rc2

### PR DESCRIPTION
**Pull request by str4d**

### CODE REVIEW

**Review by nuttycom — CHANGES_REQUESTED** [review]
This will need to be rebuilt after https://github.com/zcash/zcash/pull/6607 merges

File: zcutil/make-release.py [hunk 1]
```diff
@@ -426,12 +426,21 @@ def patch_book_release_support(release, releaseheight):
...
+                    weeks_to_eos = int(line.split(weeks_prefix)[1].split(';')[0])
+                if halt_prefix in line:
+                    val = line.split(halt_prefix)[1].split(';')[0]
+                    if val == 'APPROX_RELEASE_HEIGHT + ACTIVATION_TO_DEPRECATION_BLOCKS':
+                        halt_height = None
+                    else:
+                        halt_height = int(val)
+        if halt_height is None:
+            halt_height = releaseheight + (weeks_to_eos * 7 * 24 * 48)
+        eos_date = today + timedelta(weeks=weeks_to_eos)
```
Comment (daira) [inline zcutil/make-release.py:442]: This will get the EoS date wrong if `DEPRECATION_HEIGHT` is set explicitly. I'll file an issue.

**Review by daira — DISMISSED** [review]
utACK

**Review by nuttycom — APPROVED** [review]
utACK https://github.com/zcash/zcash/pull/6606/commits/7e105832ae1d5c28440fa63b629bfc27bd85190c

**Review by daira — APPROVED** [review]
re-utACK 7e105832ae1d5c28440fa63b629bfc27bd85190c

**Pull request merged**